tests/functional/test_adcm_upgrade.py

```diff
@@ -10,176 +10,77 @@
-        adcm_credentials=adcm_credentials,
-        adcm_tag=old_tag,
-        volumes={volume.name: {'bind': '/adcm/data', 'mode': 'rw'}},
-        init=True,
-    ) as old_adcm_client:
-        hostprovider_bundle = old_adcm_client.upload_from_fs(get_data_dir(__file__, 'hostprovider'))
-        hostprovider = hostprovider_bundle.provider_create(name=f"test_{random_string()}")
-        host = hostprovider.host_create(fqdn=f"test_host_{random_string()}")
-
-        cluster_bundle = old_adcm_client.upload_from_fs(
-            get_data_dir(__file__, 'cluster_with_service_pass_verify')
-        )
-        cluster = cluster_bundle.cluster_prototype().cluster_create(name=f"test_{random_string()}")
-
-        cluster.host_add(host)
-
-        service = cluster.service_add(name="PassChecker")
-        service_config = service.config()
-        service_config["password"] = "q1w2e3r4"
-        service.config_set(service_config)
-    with allure.step(f'Get latest adcm repo {image}'):
-        latest_repo, latest_tag = image
-    with adcm_client(
-        adcm_repo=latest_repo,
-        adcm_credentials=adcm_credentials,
-        adcm_tag=latest_tag,
-        volumes={volume.name: {'bind': '/adcm/data', 'mode': 'rw'}},
-    ) as latest_adcm_client:
-        with allure.step('Check cluster'):
-            assert (
-                len(latest_adcm_client.cluster_list()) == 1
-            ), "There is no clusters. Expecting one"
-            cluster = latest_adcm_client.cluster_list()[0]
-            assert len(cluster.service_list()) == 1, "There is no services. Expecting one"
-            service = cluster.service_list()[0]
-            assert service.action(name="check-password").run().wait() == "success"
+        sdk_client_fs.cluster_prototype().cluster_create(name=cluster_name)
+    adcm_fs.upgrade(("hub.arenadata.io/adcm/adcm", "latest"))
+    with allure.step("Check that cluster is present"):
+        assert len(sdk_client_fs.cluster_list()) == 1, "There is no clusters. Expecting one"
+        cluster = sdk_client_fs.cluster_list()[0]
+        assert cluster.name == cluster_name, "Unexpected cluster name"
+
+
+@pytest.mark.parametrize("image", old_adcm_images(), ids=repr)
+def test_pass_in_cluster_config_encryption_after_upgrade(adcm_fs: ADCM, sdk_client_fs: ADCMClient):
+    hostprovider_bundle = sdk_client_fs.upload_from_fs(get_data_dir(__file__, "hostprovider"))
+    hostprovider = hostprovider_bundle.provider_create(name=f"test_{random_string()}")
+    host = hostprovider.host_create(fqdn=f"test_host_{random_string()}")
+    sdk_client_fs.upload_from_fs(get_data_dir(__file__, "cluster_with_cluster_pass_verify"))
+    cluster_name = f"test_{random_string()}"
+    cluster = sdk_client_fs.cluster_prototype().cluster_create(name=cluster_name)
+    cluster.host_add(host)
+    cluster.config_set_diff(dict(password="q1w2e3r4"))
+    adcm_fs.upgrade(("hub.arenadata.io/adcm/adcm", "latest"))
+    with allure.step("Check that cluster is present"):
```

Comment:
Expand step title. Now it does not reflect essence of checks

Reply:
Updated.